src/CTA.WebForms2Blazor/FileConverters/ViewFileConverter.cs

```diff
@@ -1,26 +1,93 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
+using System.Text;
 using System.Threading.Tasks;
+using CTA.WebForms2Blazor.ControlConverters;
 using CTA.WebForms2Blazor.FileInformationModel;
+using CTA.WebForms2Blazor.Helpers;
+using HtmlAgilityPack;
 
 namespace CTA.WebForms2Blazor.FileConverters
 {
     public class ViewFileConverter : FileConverter
     {
-        public ViewFileConverter(string sourceProjectPath, string fullPath) : base(sourceProjectPath, fullPath)
+        private Dictionary<String, ControlConverter> ControlRulesMap; //From constructor or service
+        //Tuple contains node to be converted, parent node, and instructions to convert node in that order
+        private List<(HtmlNode Node, HtmlNode Parent, ControlConverter Rules)> NodesToBeConverted;
```

Comment:
Instead of using a `Tuple<HtmlNode, HtmlNode, ControlConverter>`, can we create a new class called `ControlConversionAction` or something similar? This could be a simple POCO class ([POCO example](https://stackoverflow.com/a/250006)). This will improve readability because at first glance, it's difficult to interpret what exactly the Tuple object means or how these objects will be used.